default/stringtables/en.txt

```diff
@@ -4628,6 +4628,16 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_BLOCKADE_TITLE
+System Blockade
+
+SYSTEM_BLOCKADE_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system, it will establish a system blockade against all empires it is at war with and that currently do not blockade the system. When a blockade is established, [[metertype METER_SUPPLY]] propagation is interrupted for the blockaded empire. Also, if the blockaded empire posesses colonies within the system, the production generated by these colonies can only be used on the planet it is generated on. Additionally, if the blockaded planets are attacked by the incoming fleet, their meter growth (e. g. ground troop regeneration) will be suspended for one turn.
```

Comment:
This last bit isn't accurate, currently if any combat occurs then some meters, specifically troops, don't grow, even if the planet isn't attacked (you can do this with a stealthed carrier), but others do, defence definitely does as planetary defences can whittle down a visible blockading fleet. Shields are gated so if the planet has no infrastructure they don't go up, but I might be misreading the code as [I don't see how it ever goes up](https://github.com/freeorion/freeorion/blob/master/default/scripting/techs/defense/BarrierShield.focs.txt#L19)—I'm terrible at embedded max/min statements, I don't see any restrictions on combat but I also don't see how it goes up for outposts.

Maybe try
> Additionally, as blockading normally initiates a combat in the system, effects that only work if no combat has occurred such as Orbital Drydocks and Troop meter growth are blocked for that turn.